.github/workflows/link-checker.yml

```diff
@@ -0,0 +1,40 @@
+# This Link Checker is run on all documentation files once per week.
+
+# references:
+# - https://github.com/lycheeverse/lychee-action
+# - https://github.com/lycheeverse/lychee
+
+name: Link Checker
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '30 8 * * 5'
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
```

Comment:
Yes, that should work.

Actually the GITHUB_TOKEN is mostly needed to prevent being rate-limited immediately when looking up github.com links.

Feel free to commit+test your suggestions.